### PR TITLE
added suggested and destructive actions

### PR DESCRIPTION
**Pull request by SeanKuehl**
Let me know if I missed any or need to fix anything.

### CODE REVIEW

**Comment by SeanKuehl** [comment]
Made requested changes. Let me know if there's anything else that needs adjusting.

File: tools/level_editor/level_editor.vala [hunk 1]
```diff
@@ -1771,15 +1771,14 @@ public class LevelEditorApplication : Gtk.Application
...
 						, "A file named `%s` already exists.\nOverwrite?".printf(_project.basename(path))
 						);
 					Gtk.Widget btn;
-					btn = md.add_button("_No", ResponseType.YES);
-					btn.get_style_context().add_class(Gtk.STYLE_CLASS_SUGGESTED_ACTION);
+					btn = md.add_button("_No", ResponseType.NO);
+					
 					md.set_default_response(ResponseType.NO);
-					btn = md.add_button("_Yes", ResponseType.YES);
-					btn.get_style_context().add_class(Gtk.STYLE_CLASS_DESTRUCTIVE_ACTION);
```
Comment (dbartolini) [inline tools/level_editor/level_editor.vala]: Why delete the button? Just revert these two lines.

File: tools/level_editor/project_browser.vala [hunk 2]
```diff
@@ -377,8 +377,7 @@ public class ProjectBrowser : Gtk.Box
 									, "Delete Folder " + (string)name + "?"
 									);
 								Gtk.Widget btn;
-								btn = md.add_button("_Cancel", ResponseType.CANCEL);
-								btn.get_style_context().add_class(Gtk.STYLE_CLASS_SUGGESTED_ACTION);
```
Comment (dbartolini) [inline tools/level_editor/project_browser.vala]: Why delete the button? Just avoid adding STYLE_CLASS_SUGGESTED_ACTION.

File: tools/level_editor/level_editor.vala [hunk 3]
```diff
@@ -2021,12 +2020,7 @@ public class LevelEditorApplication : Gtk.Application
 			, Gtk.ButtonsType.NONE
 			, "Save changes to Level before closing?"
 			);
-		Gtk.Widget btn;
-		btn = md.add_button("Close _without Saving", ResponseType.NO);
-		btn.get_style_context().add_class(Gtk.STYLE_CLASS_DESTRUCTIVE_ACTION);
-		md.add_button("_Cancel", ResponseType.CANCEL);
-		btn = md.add_button("_Save", ResponseType.YES);
-		btn.get_style_context().add_class(Gtk.STYLE_CLASS_SUGGESTED_ACTION);
```
Comment (dbartolini) [inline tools/level_editor/level_editor.vala]: Why delete? Just avoid adding ``Gtk.STYLE_CLASS_SUGGESTED_ACTION`` to "Save" button.

**Review by dbartolini — CHANGES_REQUESTED** [review]

**Comment by SeanKuehl** [comment]
Sorry about that, there was a slight misunderstanding. Let me know if this is better.

**Pull request merged**